### PR TITLE
Typescript link migration

### PR DESCRIPTION
**Pull request by ivanlori**

### CODE REVIEW

File: src/components/Button/Button.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React, { useRef, forwardRef } from "react";
 import PropTypes from "prop-types";
 import cn from "classnames";
 import Spinner from "~/components/Spinner";
-import Link from "~/components/Link";
+import Link from "../Link";
```
Comment (svedova) [inline src/components/Button/Button.js:5]: **suggestion** WDYT about keeping the import statements consistent? 

Here we import the `Spinner` with an alias and the `Link` component relatively. They are in the same location so we can import them the same way. I'd favor the one with alias as in the future if we relocate this component elsewhere we won't have to update the import statements. 

But I don't have strong objections against the relative import. In that case I'd suggest to change the `Spinner` import to `relative` import as well.

File: src/components/Link/Link.tsx [hunk 2]
```diff
@@ -1,12 +1,21 @@
...
+  tertiary: boolean,
+  rel?: string,
+  target?: string
+}
+
+const Link: FC<Props> = forwardRef<HTMLAnchorElement, Props>(
+  ({ to, children, secondary, tertiary, className, ...props }: Props, ref): ReactElement => {
     let isExternal = false;
-    let classes = className;
+    let classes: string = className;
```
Comment (svedova) [inline src/components/Link/Link.tsx:18]: Actually the `className` can be anything there. The `classnames` module accept an `object`, `array` or `string` as arguments.

**Review by svedova — CHANGES_REQUESTED** [review]
Thanks for the changes @ivanlori! Left a couple of comments for your consideration!

File: src/components/Link/Link.tsx (hunk 2 above)
Comment (ivanlori) [reply src/components/Link/Link.tsx:18]: yes, but the output of the classnames will be always a string

File: src/components/Button/Button.js (hunk 1 above)
Comment (ivanlori) [reply src/components/Button/Button.js:5]: ok, about the ~, we can figure out because now it can be handled by the alias of tsconfig

Comment (svedova) [reply src/components/Button/Button.js:5]: Personally I'm more fond of the `~` import. Especially now that with `tsconfig` the IDE can understand it :)

File: src/components/Link/Link.tsx (hunk 2 above)
Comment (svedova) [reply src/components/Link/Link.tsx:18]: That's true. I was more thinking of this use case:

```
<Link to="/apps" className={{ myClass: displayOnlyIfThisVariableIsTrue }} />
```

But I guess forcing the developer to use `classnames` while calling the component also makes sense.

**Comment by svedova** [comment]
I've implemented the minor change and merged the PR. Thanks @ivanlori for the changes!

**Pull request merged**